crates/bevy_ecs/src/storage/blob_vec.rs

```diff
@@ -268,10 +267,11 @@ const fn padding_needed_for(layout: &Layout, align: usize) -> usize {
 mod tests {
     use super::BlobVec;
     use std::{alloc::Layout, cell::RefCell, rc::Rc};
-
-    // SAFETY: The pointer points to a valid value of type `T` and it is safe to drop this value.
-    unsafe fn drop_ptr<T>(x: *mut u8) {
-        x.cast::<T>().drop_in_place()
+    // SAFETY: The pointer points to a valid array of type `[T; len]` and it is safe to drop this value.
+    unsafe fn drop_multiple<T>(x: *mut u8, len: usize) {
+        for i in 0..len as isize {
+            x.cast::<T>().offset(i).drop_in_place()
```

Comment:
Why is this not a free function in the `blob_vec` module? 

It's repeated here and as an associated function of `ComponentDescriptor `